src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_commands.py

```diff
@@ -61,22 +61,29 @@ def test_iot_hub(self, resource_group, resource_group_location):
         ])
 
         # Test 'az iot hub update'
-        property_to_update = 'properties.messagingEndpoints.fileNotifications.maxDeliveryCount'
-        updated_value = '20'
-        self.cmd('iot hub update -n {0} --set {1}="{2}"'.format(hub, property_to_update, updated_value),
+        self.cmd('iot hub update -n {0} --fnd 80 --rd 4'.format(hub),
```

Comment:
since many arguments are added into update command, could we test more arguments? like `--feedback_ttl`, `feedback_lock_duration`, `--fileupload_storage_connectionstring` and `--fileupload_storage_container_name`?

Reply:
updated